### DISCUSSION TITLE
Create github project board(s) for management

### DISCUSSION THREAD

**Discussion by yusufkandemir** [area17/twill#887] [Ideas]
You guys are sometimes saying that `x feature is on our roadmap`, `we are working on y feature` on the Spectrum chat and Github issues. Some issues have `working on it` labels but not have anyone assigned to it. It seems you are open to working with the community too.

I think it may be a good idea to create [github project board(s)](https://help.github.com/en/articles/about-project-boards) for managing this repository. Users can easily track what is being **in hold/in progress/done**, also contributors can review and help the related issues.
Individual boards for documentation, roadmap and maybe maintenance(about internals) will be nicer than a single board in my opinion.

Also please see [this article](https://opensource.com/article/18/4/keep-your-project-organized-git-repo) about managing open source project workflow.

**Comment by yanhao-li** [comment]
Thank you for the suggestions!
I also think we need to find a way to put everything on track. 
Github project may be a good option to manage this, I will review and discuss with other team members.
